.github/workflows/build.yml

```diff
@@ -7,22 +7,39 @@ jobs:
       max-parallel: 1
       matrix:
         config:
-          - { name: "Linux-x86_64-Release", type: "release", cmake_args: "-DBUILD_STEAMLIB=1 -DUSE_GRAPHICS_NRI=1" }
-          - { name: "Linux-x86_64-Debug", type: "debug", cmake_args: "-DBUILD_STEAMLIB=1 -DUSE_GRAPHICS_NRI=1" }
+          - {
+              name: "Linux-x86_64-Release",
+              type: "release",
+              cmake_args: "-DBUILD_STEAMLIB=1 -DUSE_GRAPHICS_NRI=1",
+            }
+          - {
+              name: "Linux-x86_64-Debug",
+              type: "debug",
+              cmake_args: "-DBUILD_STEAMLIB=1 -DUSE_GRAPHICS_NRI=1",
+            }
     name: ${{ matrix.config.name }}
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-20.04
     container:
-      image: ubuntu:22.04
+      image: ubuntu:20.04
       env:
         DEBIAN_FRONTEND: noninteractive
         DEBCONF_NONINTERACTIVE_SEEN: true
     steps:
-      - name: Setup
+      - name: Setup GHA caching for vcpkg - https://learn.microsoft.com/en-us/vcpkg/consume/binary-caching-github-actions-cache
         run: |
           echo "ACTIONS_CACHE_URL=$ACTIONS_CACHE_URL" >> "$GITHUB_ENV"
           echo "ACTIONS_RUNTIME_TOKEN=$ACTIONS_ID_TOKEN_REQUEST_TOKEN" >> "$GITHUB_ENV"
-          apt update
-          apt install -y git curl clang cmake build-essential libsdl2-dev libopenal-dev libvorbis-dev libfreetype6-dev libcurl4-gnutls-dev git zip unzip python3
+
+      - name: Add CMake APT repo - glslang requires newer Cmake than provided by 20.04
+        uses: gerlero/add-apt-repository@v1
+        with:
+          uri: https://apt.kitware.com/ubuntu/
+          key: https://apt.kitware.com/keys/kitware-archive-latest.asc
+
+      - name: Install deps
+        uses: gerlero/apt-install@v1
+        with:
+          packages: kitware-archive-keyring git curl clang cmake build-essential libsdl2-dev libopenal-dev libvorbis-dev libfreetype6-dev libcurl4-gnutls-dev git zip unzip python3 autoconf libtool meson automake libtool pkg-config python3-jinja2
```

Comment:
his profile seems find but im always hesitant to introduce deps on third party devs: gerlero